### ISSUE TITLE
ESLint fixes are no longer available after enabling the examples processor

### ISSUE THREAD

**Issue by lishaduck** [gajus/eslint-plugin-jsdoc#1431]
## Expected behavior
ESLint should continue to provide fixes for `.ts` files.

## Actual behavior
ESLint no longer provides any fixes for any rules in files that the processor runs on.

## ESLint Config

```js
// Format JS (or JSON) code here
```

## ESLint sample

Add this code to a config:
```ts
{
  name: "deputy/docs/examples/setup",

  plugins: {
    examples: getJsdocProcessorPlugin({
      checkDefaults: true,
      checkParams: true,
      checkProperties: true,

      exampleCodeRegex: "^```([\\s\\S]*)```\\s*$",

      parser: ts.parser as Linter.Parser,
    }),
  },
  processor: "examples/examples",
},
```

## Environment

- Node version: v24.2.0
- ESLint version v9.31.0
- `eslint-plugin-jsdoc` version: 51.3.1

**Comment by LangLangBart** [comment]
@brettz9



Would it be possible to temporarily re-enable `supportsAutofix` and ignore the `fix` for the JSDoc issues?

```diff
--- a/src/getJsdocProcessorPlugin.js
+++ b/src/getJsdocProcessorPlugin.js
@@ -544,4 +544,5 @@ export const getJsdocProcessorPlugin = (options = {}) => {
                 // suggestions: {desc: , messageId:, fix: }[],
               } = msg;
+              delete msg.fix;
 
               const [
@@ -665,6 +666,5 @@ export const getJsdocProcessorPlugin = (options = {}) => {
           return [];
         },
-        // Todo: Reenable
-        supportsAutofix: false,
+        supportsAutofix: true,
       },
     },
```


I spent some time trying to enable JSDoc checking for my `@example` tags, only to remove it again because of the issue described in this report.

I'm happy as long as JSDoc is being linted and I can see all issues, I can fix the examples by hand.
